vpc/service/assign_addresses_v3.go

```diff
@@ -668,45 +679,64 @@ func addDefaultRoute(ctx context.Context, jumbo bool, ass *vpcapi.AssignIPRespon
 			},
 		}
 	}
+
+	if ass.TransitionAssignment != nil {
+		// Transition assignment takes a slightly conservative stance
+		ass.TransitionAssignment.Routes = []*vpcapi.AssignIPResponseV3_Route{
+			{
+				Destination: "0.0.0.0/0",
+				Mtu:         1500,
+				Family:      vpcapi.AssignIPResponseV3_Route_IPv4,
+			},
+			{
+				Destination: "::/0",
+				Mtu:         1500,
+				Family:      vpcapi.AssignIPResponseV3_Route_IPv6,
```

Comment:
Why do we need v6 routes on the transition interface?

Reply:
This is laziness :(